pkg/Cpanel/Security/Advisor/Assessors/Trojans.pm

```diff
@@ -73,7 +91,618 @@ sub _check_for_libkeyutils {
     );
 
     return 1;
+}
+
+sub _check_for_UMBREON_rootkit {
+    my ($self) = @_;
+    my $dir = '/usr/local/__UMBREON__';
+    my $dir2 = '/usr/local/UMBREON';
+    if ( chdir $dir or chdir $dir2 ) {
```

Comment:
Yes, unfortunately there isn't much information available on the UMBREON rootkit.  The only thing I am able to find is where it can be purchased (not something we want to link to obviously).  We don't see this rootkit too often, so perhaps it can be removed from Security Advisor.